std/haxe/async/Defer.hx

```diff
@@ -0,0 +1,11 @@
+package haxe.async;
+
+class Defer {
+	/**
+		Schedules the given function to run during the next processing tick.
+		Convenience shortcut for `Timer.delay(f, 0)`.
+	**/
+	public static inline function nextTick(f:() -> Void):asys.Timer {
+		return asys.Timer.delay(f, 0);
```

Comment:
I know this PR is about API, not implementations. But just be aware that `haxe.Timer` on java throws runtime errors if delay is 0ms. And small delays like 1ms don't guarantee the function won't be executed on the same tick.
See travis builds for the latest two commits here: https://github.com/haxe-utest/utest/commits/master?after=61cfb1d9997eaf18ebcd6c9f4a417a832e875116+0